addons/portal_rating/models/rating_rating.py

```diff
@@ -54,3 +55,15 @@ def _synchronize_publisher_values(self, values):
             if not values.get('publisher_id'):
                 values['publisher_id'] = self.env.user.partner_id.id
         return values
+
+    def _to_store_defaults(self):
+        return super()._to_store_defaults() + (
+            [
+                Store.One("message_id", []),
+                Store.One("publisher_id", ["name", "is_company", "user", "write_date"]),
+                Store.Attr("publisher_comment"),
+                Store.Attr("publisher_datetime"),
+            ]
+            if self.publisher_id
+            else []
+        )
```

Comment:
Those don't look like "defaults", they look very specific to the flow of `/website/rating/comment` so I would just list these fields there instead (as extra_fields if defaults are necessary).
Is there any other usage? A method could be created to result this list of fields and called in the other places.